app-text/languagetool/languagetool-6.3.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
Why not EAPI 8?

Reply:
No especial reason, I guess that at the moment of opening the PR, EAPI 8 was not a thing yet¿?